community/server/src/main/java/org/neo4j/server/rest/web/CorsFilter.java

```diff
@@ -68,7 +72,22 @@ public void doFilter( ServletRequest servletRequest, ServletResponse servletResp
         HttpServletRequest request = (HttpServletRequest) servletRequest;
         HttpServletResponse response = (HttpServletResponse) servletResponse;
 
-        response.setHeader( ACCESS_CONTROL_ALLOW_ORIGIN, "*" );
+        String uri = "*";
+        if ( filterConfig != null )
+        {
+            uri = filterConfig.getInitParameter( "access_control_allow_origin" );
```

Comment:
The overhead is presumably negligible, but still - if we just passed `uri` as an argument to the constructor of this class, we wouldn't have to add this additional code on the hot path.